src/configuration.ts

```diff
@@ -96,7 +97,7 @@ export class Configuration {
 
     public subtasks: object = null;
 
-    public toolbox_map = new Map<AllowedToolboxItem, any> ([
+    public toolbox_map = new Map<AllowedToolboxItem, new (..._) => ToolboxItem>([
```

Comment:
maybe add an excerpt from your pr comment here to explain this syntax

Reply:
c1200fd